### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tanyakovchuk**
[DEMO LINK](https://taya294.github.io/react_decompose/)

### CODE REVIEW

**Review by progmari — APPROVED** [review]
Good job!

File: src/components/Article.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.article__paragraph:not(:first-child) {
+  margin-top: 0.7em;
+}
+
+@media (min-width: 500px) {
+  .header {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
```
Comment (VinnieJ-2k20) [inline src/components/Article.css:25]: Let's move these styles to `header` styles.

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```
Comment (VinnieJ-2k20) [inline src/components/Header.jsx:12]: Let's make a separate `Navigation` component.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
You are on the right way.
But, some changes still needed.

First, let's organise your `components` folder.
You should have separate folder for each component, where its `.js` and `.css` files are stored.
Second, let's move Navigation to separate component.
Also, rearrange some styles. Don't mix styles for different components in one file.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**